tools/nni_cmd/launcher.py

```diff
@@ -263,6 +263,8 @@ def set_experiment(experiment_config, mode, port, config_file_name):
         request_data['multiPhase'] = experiment_config.get('multiPhase')
     if experiment_config.get('multiThread'):
         request_data['multiThread'] = experiment_config.get('multiThread')
+    if experiment_config.get('disableLog'):
+        request_data['disableLog'] = experiment_config.get('disableLog')
```

Comment:
I think 'disableLog' is very confusing. User doesn't know what is it used to, disable NNIManager log, disable all trials log? 
Actually, it just disable posting trial logs back to nnimanager. 

I STRONGLY suggest to use a meaning name for this option flag

Reply:
disableLog is removed.